agent/consul/server.go

```diff
@@ -16,6 +16,8 @@ import (
 	"sync"
 	"time"
 
+	"sync/atomic"
```

Comment:
This should get moved up after `sync` w/o the extra space (tools like `goimports` will keep these sorted for you).